packages/mdc-dialog/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MDCComponent} from '@material/base';
+import MDCDialogFoundation from './foundation';
+import * as util from './util';
+
+export {MDCDialogFoundation};
+
+export class MDCDialog extends MDCComponent {
+  static attachTo(root) {
+    return new MDCDialog(root);
+  }
+
+  get open() {
+    return this.foundation_.isOpen();
+  }
+
+  set open(value) {
+    if (value) {
+      this.foundation_.open();
+    } else {
+      this.foundation_.close();
+    }
+  }
+
+  get acceptButton() {
```

Comment:
should these 3 getters be private? Usually we don't want to expose inner elements of components. Also, let's call `dialog` `dialogSurface` instead.